src/connectors/ona-api/aggregateFormData.js

```diff
@@ -165,6 +181,22 @@ function processFormData(formData, indicatorField, aggregateOptions) {
         // reduce sumTotal for current groupData
         for (let x = 0; x < groupData.length; x += 1) {
           sumTotal += parseInt(groupData[x][indicatorField] || 0, 10);
+          const { parsedUID } = groupData[x];
+          parsedLocName = parsedUID;
```

Comment:
What is this being used for? Can we just pass in `parsedUID` as is into `currentPeriodaggregatedDataObj`?

Reply:
we are getting the `parsedUID` from the groupData which is not exposed when building the  `currentPeriodaggregatedDataObj` which is outside the `groupData` loop so we are assigning it to a var which is defined outside that loop.

Reply:
👍